docs/docs/getting-started/set-server-time.md

```diff
@@ -1,13 +1,14 @@
 ---
 title: Setting the Server Time
-layout: article
-category: install
+sidebar_position: 1
```

Comment:
I don't think this has to be in first position. Install and configuration should be first, this could go in an advanced config section or such.

Reply:
Why did you mark this as resolved?